satellite/satellitedb/orders.go

```diff
@@ -5,25 +5,30 @@ package satellitedb
 
 import (
 	"context"
+	"database/sql"
+	"time"
 
 	"github.com/golang/protobuf/ptypes"
 	"github.com/zeebo/errs"
 
 	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
 	dbx "storj.io/storj/satellite/satellitedb/dbx"
 )
 
+const defaultIntervalSeconds = 60 * 60 // hour
```

Comment:
A possible alternative for counting the number of seconds in an hour is: `int(time.Hour / time.Second)`.

Reply:
it looks that I can do `int(time.Hour.Seconds())`

Reply:
eh but then it cannot be constant - `const initializer int(time.Hour.Seconds()) is not a constant`

Reply:
You can, but only as a variable, not as a constant.

Reply:
`int(time.Hour / time.Second)` is still a constant, because it is an expression of two constants.

Reply:
If you do it like `var defaultIntervalSeconds = int(time.Hour.Seconds())` then you don't need the `int()` typecast as `Seconds()` return type is `int`. I think there will be a linter to fail because of this.

Reply:
`Seconds()` returns float64 but I used your version after all